content/influxdb/v2.2/reference/release-notes/influxdb.md

```diff
@@ -8,6 +8,131 @@ menu:
 weight: 101
 ---
 
+## v2.2 [2022-04-04]
+
+This release includes the following new [features](#features) and several [bug fixes](#bug-fixes).
+
+### Features
+
+- [Technical preview: replicate data remotely](#technical-preview-replicate-data-remotely)
+- [Flux updates](#flux-updates)
+- [Build maintenance](#build-maintenance)
+- [Task metadata](#task-metadata)
+- [Metrics](#metrics)
+- [Display runtime configuration in use by `influxd`](#display-runtime-configuration-in-use-by-influxd)
+- [Recover user credentials](#recover-user-credentials)
+- [Security updates](#security-updates), including a new `hardening-enabled` option to enable additional security.
+
+#### Technical preview: replicate data remotely
+
+- Add the option to [Replicate data from InfluxDB OSS to InfluxDB Cloud](/influxdb/v2.2/write-data/replication).
+
+  This [**technical preview**](/influxdb/v2.1/reference/glossary/#technical-preview) feature lets you create a durable subscription on a per bucket basis for the purposes of replicating data on-write from an InfluxDB OSS instance to InfluxDB Cloud.
+  - Lets you store, analyze, and aggregate data locally while also forwarding newly arriving data to a centralized InfluxDB Cloud account.
+  - If InfluxDB OSS loses connectivity to InfluxDB Cloud, the data gets sent when network connectivity is reestablished.
+  - Configuration options include a maximum buffer size and data age restrictions to restrict the amount of data stored on disk.
+
+#### Flux updates
+
+- Update to [Flux v0.150.0](/flux/v0.x/release-notes/#v01500-2022-01-19).
+- Add option to log Flux queries that are cancelled because of server shutdown.
+
+#### Build maintenance
+
+- Upgrade `protobuf` library.
+
+#### Task metadata
+
+- Add option to pass `type=basic` to [`/api/v2/tasks`](/influxdb/v2.1/api/#tag/Tasks) to return task metadata without the query text.
+
+#### Metrics
+
+- Add Prometheus metrics for TSM compactions, including:
```

Comment:
not sure that "Prometheus" is meaningful here.  They are Prometheus Exposition Format....   But the key point is that these are key metrics about the storage sub-system which are exposed now by InfluxDB....

Most of these were previously available with InfluxDB v1, but were not initially available with InfluxDB v2.  These are being re-introduced to assist with troubleshooting and other operational management.

Reply:
Still at it!